includes/Admin/Cross_Origin_Isolation.php

```diff
@@ -310,12 +310,6 @@ protected function is_needed() {
 			return false;
 		}
 
-		// Backwards compatible, default values added in WP 5.5.
-		$check = get_user_meta( $user_id, Preferences::MEDIA_OPTIMIZATION_META_KEY, false );
-		if ( empty( $check ) ) {
-			return true;
-		}
-
-		return rest_sanitize_boolean( $check[0] );
+		return get_user_meta( $user_id, Preferences::MEDIA_OPTIMIZATION_META_KEY, true );
```

Comment:
Should this have `(bool) get_user_meta(...)`?

Reply:
I added back the `rest_sanitize_boolean` function call.